.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,14 @@
+### Problem description
+
+
+
+### Screenshot / Reference 
+
+
+
+### Checklist
+- [ ] I've set a rough estimate | Estimate is TBA
+- [ ] I've assigned the issue with a label (whatever is most relevant)
+- [ ] I've considered assigning the issue with a person | No assignment needed, just making a note
+- [ ] I've added the issue to a milestone | N/A for this milestone
```

Comment:
This is more  a sprint planning thing too...

Reply:
So remove the checklist entirely? I think assigning a label if nothing else is ok...But I see where you're coming from, this should all be defined sprint per sprint...

Reply:
@AlexGilleran Nah actually you're right, the whole checklist should be done in sprint planning, I'll remove